### PR TITLE
Download Ragel binary

### PR DESCRIPTION
**Pull request by peter-edge**
This downloads Ragel as a binary from https://github.com/yarpc/ragel instead of compiling from source. This saves about 1.5 mins on Travis CI for each of 3 of the 4 builds.

### CODE REVIEW

**Comment by breerly** [comment]
This looks promising - can you update the PR description with the results?

File: etc/make/deps.mk [hunk 1]
```diff
@@ -21,15 +21,17 @@ EXTRA_GO_BIN_DEPS = \
 GLIDE_VERSION := 0.12.3
 THRIFT_VERSION := 1.0.0-dev
 PROTOC_VERSION := 3.3.0
-RAGEL_VERSION := 6.10
+RAGEL_VERSION := 0.1.0
```
Comment (breerly) [inline etc/make/deps.mk]: If this works, then we should release a `6.10` and kill `0.1.0`.

File: etc/make/deps.mk [hunk 2]
```diff
@@ -53,22 +55,20 @@ PROTOC_LIB = $(LIB)/protoc-$(PROTOC_VERSION)
...
 RAGEL = $(BIN)/ragel
 
 GEN_BINS = $(THRIFT) $(PROTOC) $(RAGEL)
 EXTRA_BINS = $(GLIDE)
 
-$(RAGEL_TAR):
+$(RAGEL_BIN):
 	@mkdir -p $(RAGEL_LIB)
-	curl -L "https://www.colm.net/files/ragel/ragel-$(RAGEL_VERSION).tar.gz" > $(RAGEL_TAR)
+	curl -L "https://github.com/yarpc/ragel/releases/download/v$(RAGEL_VERSION)/ragel-$(RAGEL_OS)-$(RAGEL_ARCH)" > $(RAGEL_BIN)
```
Comment (breerly) [inline etc/make/deps.mk]: nice

**Comment by breerly** [comment]
Just popped into my head: instead of this approach, Is it possible to make ragel a local-only dependency by checking in the files it generates?

**Comment by kriskowal** [comment]
🎉

**Pull request merged**